### PR TITLE
Match HTML and CSS

### PR DESCRIPTION
**Pull request by Amit1Paz**
I created the part that appears after the user's selection.
The "placeholder" in the comment will appear before the user sees the computer's move.
The "winner" class will be assigned to the winner's side only after the winner will be announced.

I commented out the div "choose-container" to create this part.  The "choose-container" and the "match-container" will be visible alternately.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -75,6 +75,7 @@ h2 {
 }
 
 /***** main *****/
+/***** choose *****/
```
Comment (Sa-ar) [inline style.css:82]: What do you think about splitting this huge CSS file into more files with different purposes?

File: index.html [hunk 2]
```diff
@@ -19,14 +19,29 @@ <h2 class="score-box__score">0</h2>
     </div>
   </header>
   <main>
-    <div class="choose-container">
+    <!-- <div class="choose-container transparent">
```
Comment (Sa-ar) [inline index.html]: I don't see why do you need to add this comment or add a transparent class to it.

File: style.css [hunk 3]
```diff
@@ -129,10 +171,34 @@ footer {
 
 
 /***** media *****/
-@media (max-width: 665px) {
+@media (max-width: 750px) {
+    .winner {
+        box-shadow: inset 0 5px 1px rgb(196, 196, 224), 0 0 1px 1.5rem rgba(255, 255, 255, 0.1), 0 0 1px 3rem rgba(255, 255, 255, 0.1), 0 0 1px 4.5rem rgba(255, 255, 255, 0.1);
```
Comment (Tamir198) [inline style.css]: I would save rgba(255, 255, 255, 0.1) and other identical colors in a variable, so if you need to change this you will only change the variable value.

**Review by Tamir198 — COMMENTED** [review]
I think you did nice work, good work on making it responsive.
I would try to use more variables for repeating values in your css.

File: style.css (hunk 3 above)
Comment (Amit1Paz) [reply style.css]: Fixed.

**Review by Sa-ar — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (Amit1Paz) [reply style.css:82]: I think its a good idea. I will split the css file into smaller files.

File: index.html (hunk 2 above)
Comment (Amit1Paz) [reply index.html]: The transparent class will help me disappear elements easily with js. The comment is actually unneeded so i will  delete it.

**Pull request merged**